cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityKnownLimitationsTest.scala

```diff
@@ -1,49 +0,0 @@
-/*
- * Copyright (c) "Neo4j"
- * Neo4j Sweden AB [http://neo4j.com]
- *
- * This file is part of Neo4j.
- *
- * Neo4j is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-package org.neo4j.cypher.docgen
-
-import org.neo4j.cypher.docgen.tooling._
-
-class SecurityKnownLimitationsTest extends DocumentingTest with QueryStatisticsTestSupport {
-  override def outputPath = "target/docs/dev/ql/administration/security/"
-
-  override def doc: Document = new DocBuilder {
-    doc("Known limitations of security", "administration-security-limitations")
```

Comment:
Generated to

neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-limitations.adoc